### PR TITLE
README with new repo structure approach

### PR DESCRIPTION
**Pull request by jframos**
#### Reviewers

@jesuspg @hmunfru @flopezag 
#### Description

Approach on repo organization:
- Only one version for recipes and manifests will be available for each GE/GEi. The same as the current SVN. Tags in the repo will have previous versions.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -7,58 +7,103 @@ The structure of the directory looks something like this:
...
+  and we create a new tag with all stable content of the repo for that FIWARE release.
 - develop, default work branch.
-- feature/<GEri/GEi-name>, branch used to the GEri/GEi owner to upload the content. When you consider that the scripts are correct you should launch a pull request to develop. FIWARE Lab team will check the scripts in order to secure that it is working and if all is correct we will accept the pull request to develop branch.
\ No newline at end of file
+- feature/<GEri/GEi-name>, branch used by the GEri/GEi owner to upload the content.
+
+How to contribute to this repo
+==============================
+
+Probably, you will not have access rights to create branches in this repo. If so, you must request them to FIWARE Lab team
```
Comment (flopezag) [inline README.rst]: I will redraw the option of request access to the repo and allow only the management through PR from a fork repo.

Comment (jframos) [inline README.rst]: Ok, feel free to add in this branch whatever you consider necessary.

Comment (flopezag) [inline README.rst]: Maybe, there is a missunderstanding, I suggest to say something like it:

```
"Probably, you will not have access rights to create branches in this repo. We do not give access to other developers except the FIWARE Lab support Team, you should work in your own forked repository and make pull request to our repository."
```

the rest of description should reflect this idea if we do not want to give access to external developers inside this repo.

Comment (jframos) [inline README.rst]: :+1:

**Comment by flopezag** [comment]
LGTM

**Comment by hmunfru** [comment]
LGTM

**Comment by jesuspg** [comment]
LGTM

**Pull request merged**